src/firefly/js/ui/dynamic/DynComponents.jsx

```diff
@@ -206,28 +206,29 @@ export function PositionAndPolyFieldEmbed({fieldDefAry, plotId, toolbarHelpId, i
 
     return (
         <EmbeddedPositionSearchPanel {...{
-            toolbarHelpId,
-            WrapperComponent,
-            hipsUrl, hipsFOVInDeg,
-            initCenterPt,
-            mocList, sRegion,
-            coordinateSys: csysStr,
-            targetKey,
-            polygonKey,
-            sizeKey,
             initSelectToggle: initToggle,
             usePosition,
             usePolygon,
-            plotId,
-            minValue, maxValue, searchAreaInDeg,
             targetPanelExampleRow1, targetPanelExampleRow2,
-            polygonExampleRow1:polyType?.targetDetails?.targetPanelExampleRow1,
-            polygonExampleRow2:polyType?.targetDetails?.targetPanelExampleRow2,
             nullAllowed,
             insetSpacial,
-            otherComponents,
-            doSearch:submitSearch
-            }}/>
+            slotProps: {
+                hipsTargetView: {
+                    mocList, sRegion, plotId, toolbarHelpId, hipsUrl, hipsFOVInDeg, initCenterPt,
+                    coordinateSys: csysStr,
+                },
+                targetPanel: { targetKey, targetPanelExampleRow1, targetPanelExampleRow2, },
+                sizeInput: {min:minValue,max:maxValue,sizeKey, initValue: searchAreaInDeg},
+                polygonField: {
+                    polygonKey,
+                    polygonExampleRow1:polyType?.targetDetails?.targetPanelExampleRow1,
+                    polygonExampleRow2:polyType?.targetDetails?.targetPanelExampleRow2,
+                },
+                formPanel: slotProps.FormPanel ?? {}
```

Comment:
Just realized after examining the code.  'is required' is only in the comment.
```note- onSuccess is required for this panel to function like a FormPanel```
In the above code, it would render a FormPanel without a `onSucess`.  Is this what you wanted?

Reply:
Good catch, thanks